x-pack/plugins/ml/common/types/field_histograms.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface NumericDataItem {
+  key: number;
+  key_as_string?: string | number;
+  doc_count: number;
+}
+
+export interface NumericChartData {
+  data: NumericDataItem[];
+  id: string;
+  interval: number;
+  stats: [number, number];
+  type: 'numeric';
+}
+
+export const isNumericChartData = (arg: any): arg is NumericChartData => {
```

Comment:
shall we use `unknown` instead of `any` for type guards?

Reply:
It's a good idea, but since we used `any` so far for all type guards, I'd rather do this in a follow up and change it for all type guards we have if we want to do it. There might be some changes necessary to how we check objects.